EIPS/eip-XXX.md

```diff
@@ -0,0 +1,415 @@
+---
+eip: TODO
+title: Ethereum state using a unified binary tree
+author: Vitalik Buterin (@vbuterin), Guillaume Ballet (@gballet), Dankrad Feist (@dankrad), Ignacio Hagopian (@jsign), Kevaundray Wedderburn (@kevaundray), Tanishq Jasoria (@tanishqjasoria), Gajinder Singh (@g11tech), Danno Ferrin (@shemnon), Piper Merriam (@pipermerriam), Gottfried Herold (@GottfriedHerold)
```

Comment:
This EIP pulls many ideas from EIP-6800, so I'd still honor the work of those authors in this EIP.

If anyone wants their name to be removed, please LMK!